app/views/blog_posts/index.html.erb

```diff
@@ -14,6 +14,8 @@
         <% end %>
       </div>
     <% end %>
+    <% @blog_posts = BlogPost.page(params[:page]) %>
```

Comment:
`@blog_posts` is already set in the controller. You don't need to set it again in the view.